src/app/dashboard/_components/smart-notes/CommandMenu.tsx

```
@@ -15,153 +21,157 @@ export interface Command {
           e.preventDefault();
-          setSelectedIndex(i => {
-            const newIndex = (i + 1) % filteredCommands.length;
+setSelectedIndex(i => {
+             if (filteredCommands.length === 0) return 0;
+             const newIndex = (i + 1) % filteredCommands.length;
             scrollSelectedIntoView(newIndex);
-            return newIndex;
-          });
-          break;
+             return newIndex;
+           });
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)